sw/device/silicon_creator/lib/drivers/hmac_functest.c

```diff
@@ -0,0 +1,64 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/arch/device.h"
+#include "sw/device/lib/base/memory.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/runtime/print.h"
+#include "sw/device/lib/testing/test_main.h"
+#include "sw/device/silicon_creator/lib/drivers/hmac.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+const char kGettysburgPrelude[] =
```

Comment:
Make sure to mark your constants as static.

Reply:
Done.